src/Routes.js

```diff
@@ -29,6 +32,7 @@ export const Routes = (props) => {
     return (
         <Switch>
             <InsightsRoute path={ paths.general } component={ General } rootClass='general'/>
+            <InsightsRoute exact path={ paths.applications } component={ Applications } rootClass='applications'/>
 
             { /* Finally, catch all unmatched routes */ }
             <Route render={ () => some(paths, p => p === path) ? null : (<Redirect to={ paths.general }/>) }/>
```

Comment:
My best guess is that this is what's causing the aforementioned issue, but I don't have any clue...

Reply:
I think the problem is with this function https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/src/helpers.js#L70

I'll write a quick fix for it.

Reply:
Oh, great! Thanks a lot!

Reply:
This should do the trick https://github.com/RedHatInsights/frontend-components/pull/265

Reply:
Thanks a lot! Works like a charm! :star2: